### DISCUSSION TITLE
Can the FLORIS model be applied to very large wind farms

### DISCUSSION THREAD

**Discussion by styxmshy** [NatLabRockies/floris#260] [Q&A] [answered]
Large wind farms which are dominated by the "deep array" effect act as entities. Single neighbor to neighbor relations is no longer relevant for the overall behavior of the farm.
Q1: Whether the FLORIS model belongs to the "bottom-up" type?  Simply spoken, the "bottom-up" approach for wind farms consists of an overlay or a superposition of wakes of the single turbines.
Q2: Can the FLORIS model be applied to very large wind farms (10 by 10 turbines and more within one wind farm)?

**Comment by tatesnow** [comment]
Hi @styxmshy,
FLORIS can be applied to large wind farms, I have used FLORIS on Horns Rev wind farm (8*10 wind turbine)

**Reply by styxmshy** [reply]
Thank you for answer!  Best, Shu Tong.

**Reply by CSUchenenji** [reply]
Hello, I'm a beginner. Can you share the source code of Floris on Horns Rev wind farm (8*10 wind turbine)

**Comment by styxmshy** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我将尽快给您回复。

**Comment by Bartdoekemeijer** [comment, marked as answer]
Hi,

Please see my response to Issue #276. In short, yes, FLORIS can be used for large farms. We have validated the FLORIS model with an offshore wind farm of 111 wind turbines. Currently, we have seen that the Gaussian model's accuracy diminishes for turbines very deep in a large offshore wind farm. The anticipated new version of FLORIS, expected to be released in 1-2 months, should address those issues.

I hope this helps. Let me know if anything remains unclear.

Best,
Bart

**Reply by styxmshy** [reply]
Thank you for the answer, Bart. I look forward to the future version of FLORIS! Best, ShuTong.

**Comment by styxmshy** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我将尽快给您回复。
